### ISSUE TITLE
feat: Export to .pdf and .docs

### ISSUE THREAD

**Issue by bakikucukcakiroglu** [steven-tey/novel#430]
### Describe the feature you'd like to request

I think export to pdf and docs is a necessity. 

Note: I started to implemented a project using novel and I've just noticed that it is not possible to export as pfd or docs. If I cannot find a solution I will probably need to start from scratch using a different tool. So any help or advice is very welcomed at the moment. 

### Describe the solution you'd like to see

.

**Comment by shakedhagag** [comment]
I agree! there is a tiptap extension that can export to certain doc types like docx markdown etc. but no pdf.
We can use a markdown2pdf package for that but I dont know if that is the best approach:
https://github.com/realdennis/md2pdf

**Comment by bakikucukcakiroglu** [comment]
Built-in support for various export types would also give a competitive advantage.

**Comment by Jkense** [comment]
Agreed, was just writing a features request on the same topic. Would be amazing.

I am not too familiar with Novel, other than that it's based on Tiptap. If you follow their [docs](https://tiptap.dev/docs/guides/output-json-html) you'll see that they reccomend to export in different formats than MDX. Following their advice a different export might be better (from which doc and pdf can be converted).

**Comment by bakikucukcakiroglu** [comment]
Do not waste your time writing a feature request. This library is dead. You can use tiptap notion like template. It also enables using pro features for free with a free tiptap account.
